00-max.js

```diff
@@ -1,3 +1,4 @@
 export function max(array) {
-    // TODO: implement max function
+  if (!array || array.length === 0) return undefined;
```

Comment:
If you flipped that condition, you could save a line, because `return undefined` is the same as not returning anything.